### DISCUSSION TITLE
Suppressing traffic or alerts from CVE/Vulnerability scanner?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [Security-Onion-Solutions/securityonion#4826] [Unsupported Versions] [answered]
What's the best method for supressing the millions of alerts in SO caused by our vulnerability scanners? BPF rules on sensors? Threshold rules that match all SIDS for certain IPs?

What about removing current alerts from DB?

Running 2.3.60 with seperate nodes for management, log, and search.

**Comment by ben-sec** [comment, marked as answer]
Hi lonelyadmin,

you have to setup BPF rules for steno, zeek and suricata in /opt/so/saltstack/local/pillar/manager.sls (append at the end of the file) in order to skip the whole traffic that is comming from your vulnerability scanner:

steno:
  bpf:
    - "!(src host IP-of-your-scanner)"
zeek:
  bpf:
    - "!(src host IP-of-your-scanner)"
nids:
  bpf:
    - "!(src host IP-of-your-scanner)"

If necessary, you can do that more granualr by also defining destination hosts or networks, ports, etc. Note: If you add multiple rules, you have to add "&&" at the end each line (but the last one):

- "!(src host IP-of-your-scanner) &&"
- "another rule"

I found this page with [useful BPF examples](https://www.ibm.com/docs/en/qsip/7.4?topic=queries-berkeley-packet-filters).

Cheers, Ben
